vars/executeTests.groovy

```diff
@@ -46,4 +46,17 @@ def call(Map<String, String> config = [:]) {
         }
     }
 
+    // This executes scripts, should be under test -> scripts(extension of playbooks) in contra.yml and location(path) specified
+    if ( configData.tests.scripts ) {
+        configData.tests.scripts.each { LinkedHashMap script ->
+            HashMap scriptParams = script.vars ?: [:]
+            if (config.vars){
+                scriptParams  << (config.vars as HashMap)
+            }
+            String paramString = scriptParams ? "-${scriptParams.entrySet().iterator().join(' ')}" : ""
```

Comment:
Just a thought, you might consider returning this as ```null``` in the default case, so that when passed to ```executeInShell()``` you could use an if/else clause based on whether paramString was falsey or not. Not required, by any means, however.